フロント作成　pushしそびれたものを追加

yukri1030 opened the pull request:
# WHAT
度々すみません。以前ファイルをみていただいてLGTMをいただきました。（以下３つです）

index.html.haml
index.scss
_right.scss

その際、プッシュしそびれたファイルがあります。そのため、再度別ブランチより、プルリクエストさせていただきました。

# WHY
アプリ開発に必要なビューの実装

yukri1030 committed:
[WPI]フロント作成　pushしそびれたものを追加

# WHAT
度々すみません。以前ファイルをみていただいてLGTMをいただきました。（以下３つです）

index.html.haml
index.scss
_right.scss

その際、プッシュしそびれたファイルがあります。そのため、再度別ブランチより、プルリクエストさせていただきました。

# WHY
アプリ開発に必要なビューの実装

miyagit commented on app/views/layouts/application.html.haml:
```diff
@@ -0,0 +1,11 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
```
ハッシュロケットを直しましょう。

miyagit commented on app/views/layouts/mailer.html.haml:
```diff
@@ -0,0 +1,8 @@
+\!!!
+\%html
```
mailer系のファイルを削除しましょう。

yukri1030 committed:
フロントビューの実装

# WHAT
applicationx.html.hamlのハッシュロケットを無くしました

また、mailer.html.htmlのファイルを削除しました

# WHY
より新しい書き方に慣れるため
現在、mailerのファイルは必要ないため

miyagit commented on app/views/layouts/application.html.haml:
```diff
@@ -1,10 +1,11 @@
 %html
   %head
-    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
+    %meta = {content: "text/html;" charset: "UTF-8", http: {equiv: {Content-Type}}
```
meta=ですか？？

yukri1030 committed:
フロントの実装

# WHAT
ハッシュロケットを無くし、よりシンプルな記法に書き換えました

# WHY
みやすいため、可読性を上げるため

TOMARUMARU commented on config/routes.rb:
```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root "messages#index"
+  get 'messages',to: 'messages#index'
```
root_pathにあるので不要です

yukri1030 committed:
フロントの実装

# WHAT
ルーティングを編集しました

# WHY
不要なコードを削除し、可読性を上げるため

TOMARUMARU commented on app/views/layouts/application.html.haml:
```diff
@@ -0,0 +1,12 @@
+%html
+  %head
+    %meta {content: "text/html;" charset: "UTF-8", http: {equiv: {Content-Type}}
+    -# :content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
```
不要なコメントアウトは消しましょう

TOMARUMARU commented on app/views/layouts/application.html.haml:
```diff
@@ -0,0 +1,12 @@
+    = javascript_include_tag 'application', data: { turbolinks: { track: true }}
+    = csrf_meta_tags
+  %body
+    -# = render partial: 'layouts/flash'
```
こちらは使用予定のコメントアウトですか？

TOMARUMARU commented on app/views/layouts/mailer.text.haml:
```diff
@@ -0,0 +1 @@
+\= yield
```
mailerファイル消しましょう

yukri1030 committed:
フロントの実装

# WHAT
不要なコメントアウトを削除した

# WHY
可読性を上げるため

yukri1030 committed:
フロントの実装

# WHAT
不要なファイルを削除した

# WHY
可読性を上げるため

i-date reviewed (approved):
お疲れ様でした、LGTMです。

ご参考までに。
レビューを依頼する際には、[WIP]は外してからレビューを依頼するといいでしょう^^
WIP = 作業中 なので、一通り作業が完了→WIP外す→レビュー依頼、という流れになります！
